src/main/java/com/hipo_delivery_be/controller/HealthCheckController.java

```diff
@@ -0,0 +1,13 @@
+package com.hipo_delivery_be.controller;
+
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+public class HealthCheckController {
+    @RequestMapping(value = "/api/helloworld")
```

Comment:
@GetMapping

Reply:
@caominhnhut Changed to @GetMapping on new commit